Dockerfile.alpine

```diff
@@ -0,0 +1,59 @@
+#
+# Build Image
+#
+
+FROM alpine:3.8 as builder
+
+# Add the git repo
+ADD . /git/mapcrafter
+
+# Dependencies needed for building Mapcrafter
+# (not sure how many of these are actually needed)
+RUN apk add \
+        cmake \
+        gcc \
+        make \
+        g++ \
+        zlib-dev \
+        libpng-dev \
+        libjpeg-turbo-dev \
+        boost-dev \
+        wget \
+        python \
+        imagemagick
+
+# Textures
+# With thanks to https://mcversions.net/ for the jar link
+RUN cd /git/mapcrafter && \
```

Comment:
The new Mapcrafter doesn't need textures anymore, only the block images. This can be removed.